dmutils/forms/fields.py

```diff
@@ -46,6 +46,12 @@
 class DMBooleanField(DMFieldMixin, wtforms.fields.BooleanField):
     widget = DMCheckboxInput(hide_question=True)
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # digitalmarketplace-frontend-toolkit suffixes the id with `-1`
```

Comment:
Am I correct that we may need to revisit this when we migrate checkboxes?

Reply:
govuk-frontend v2 follows the same pattern. We would need to revisit it for govuk-frontend v3, but that could just mean overriding the property `id` for the relevant form.